selene_sdk/samplers/random_positions_sampler.py

```diff
@@ -248,17 +257,19 @@ def _retrieve(self, chrom, position):
         retrieved_seq = \
             self.reference_sequence.get_encoding_from_coords(
                 chrom, window_start, window_end, strand)
+
         if retrieved_seq.shape[0] == 0:
             logger.info("Full sequence centered at {0} position {1} "
                         "could not be retrieved. Sampling again.".format(
                             chrom, position))
             return None
-        elif np.sum(retrieved_seq) / float(retrieved_seq.shape[0]) < 0.60:
+        elif np.mean(retrieved_seq==0.25) >0.30: 
             logger.info("Over 30% of the bases in the sequence centered "
                         "at {0} position {1} are ambiguous ('N'). "
                         "Sampling again.".format(chrom, position))
             return None
 
+        #TODO: is this ok to remove now?
```

Comment:
nooo idea haha, I guess when we see it it's probably a hardware issue rather than a code issue? So maybe we can remove

Reply:
Let's remove it for now then? I don't remember seeing it ever. Is it an error that is obvious the catch if it happens again?